diff -ru /Applications/Xonotic/source/qcsrc/common/notifications/all.inc qcsrc/common/notifications/all.inc
--- /Applications/Xonotic/source/qcsrc/common/notifications/all.inc	2023-06-12 09:56:56
+++ qcsrc/common/notifications/all.inc	2026-08-28 01:33:29
@@ -111,7 +111,8 @@
     MSG_ANNCE_NOTIF_TEAM(NUM_TEAM_1, prefix##_RED, prefix, defaultvalue, sprintf(sound, strtolower(STATIC_NAME_TEAM_1)), channel, volume, position) \
     MSG_ANNCE_NOTIF_TEAM(NUM_TEAM_2, prefix##_BLUE, prefix, defaultvalue, sprintf(sound, strtolower(STATIC_NAME_TEAM_2)), channel, volume, position) \
     MSG_ANNCE_NOTIF_TEAM(NUM_TEAM_3, prefix##_YELLOW, prefix, defaultvalue, sprintf(sound, strtolower(STATIC_NAME_TEAM_3)), channel, volume, position) \
-    MSG_ANNCE_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, sprintf(sound, strtolower(STATIC_NAME_TEAM_4)), channel, volume, position)
+    MSG_ANNCE_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, sprintf(sound, strtolower(STATIC_NAME_TEAM_4)), channel, volume, position) \
+    MSG_ANNCE_NOTIF_TEAM(NUM_TEAM_5, prefix##_GREEN, prefix, defaultvalue, sprintf(sound, strtolower(STATIC_NAME_TEAM_5)), channel, volume, position)
 
 // MSG_ANNCE_NOTIFICATIONS
     MSG_ANNCE_NOTIF(ACHIEVEMENT_AIRSHOT,        N_GNTLOFF, "airshot",           CH_INFO, VOL_BASEVOICE, ATTEN_NONE, ANNCE_DEFTIME)
@@ -228,7 +229,8 @@
     MSG_INFO_NOTIF_TEAM(NUM_TEAM_1, prefix##_RED, prefix, defaultvalue, strnum, flnum, args, hudargs, multiteam_info_sprintf(icon, strtolower(STATIC_NAME_TEAM_1)), TCR(normal, type, 1), TCR(gentle, type, 1)) \
     MSG_INFO_NOTIF_TEAM(NUM_TEAM_2, prefix##_BLUE, prefix, defaultvalue, strnum, flnum, args, hudargs, multiteam_info_sprintf(icon, strtolower(STATIC_NAME_TEAM_2)), TCR(normal, type, 2), TCR(gentle, type, 2)) \
     MSG_INFO_NOTIF_TEAM(NUM_TEAM_3, prefix##_YELLOW, prefix, defaultvalue, strnum, flnum, args, hudargs, multiteam_info_sprintf(icon, strtolower(STATIC_NAME_TEAM_3)), TCR(normal, type, 3), TCR(gentle, type, 3)) \
-    MSG_INFO_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, strnum, flnum, args, hudargs, multiteam_info_sprintf(icon, strtolower(STATIC_NAME_TEAM_4)), TCR(normal, type, 4), TCR(gentle, type, 4))
+    MSG_INFO_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, strnum, flnum, args, hudargs, multiteam_info_sprintf(icon, strtolower(STATIC_NAME_TEAM_4)), TCR(normal, type, 4), TCR(gentle, type, 4)) \
+    MSG_INFO_NOTIF_TEAM(NUM_TEAM_5, prefix##_GREEN, prefix, defaultvalue, strnum, flnum, args, hudargs, multiteam_info_sprintf(icon, strtolower(STATIC_NAME_TEAM_5)), TCR(normal, type, 5), TCR(gentle, type, 5))
 
 // MSG_INFO_NOTIFICATIONS
     MSG_INFO_NOTIF(CHAT_DISABLED,                           N_CHATCON,  0, 0, "", "",       "",     _("^F4NOTE: ^BGChat is currently disabled on this server"), "")
@@ -536,7 +538,8 @@
     MSG_CENTER_NOTIF_TEAM(NUM_TEAM_1, prefix##_RED, prefix, defaultvalue, strnum, flnum, args, cpid, durcnt, TCR(normal, type, 1), TCR(gentle, type, 1)) \
     MSG_CENTER_NOTIF_TEAM(NUM_TEAM_2, prefix##_BLUE, prefix, defaultvalue, strnum, flnum, args, cpid, durcnt, TCR(normal, type, 2), TCR(gentle, type, 2)) \
     MSG_CENTER_NOTIF_TEAM(NUM_TEAM_3, prefix##_YELLOW, prefix, defaultvalue, strnum, flnum, args, cpid, durcnt, TCR(normal, type, 3), TCR(gentle, type, 3)) \
-    MSG_CENTER_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, strnum, flnum, args, cpid, durcnt, TCR(normal, type, 4), TCR(gentle, type, 4))
+    MSG_CENTER_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, strnum, flnum, args, cpid, durcnt, TCR(normal, type, 4), TCR(gentle, type, 4)) \
+    MSG_CENTER_NOTIF_TEAM(NUM_TEAM_5, prefix##_GREEN, prefix, defaultvalue, strnum, flnum, args, cpid, durcnt, TCR(normal, type, 5), TCR(gentle, type, 5))
 
 // MSG_CENTER_NOTIFICATIONS
     MSG_CENTER_NOTIF(ALONE,                             N_ENABLE,    0, 0, "",               CPID_Null,              "0 0",  _("^F4You are now alone!"), "")
@@ -823,7 +826,8 @@
     MSG_MULTI_NOTIF(prefix##_RED, defaultvalue, anncepre##_RED, infopre##_RED, centerpre##_RED) \
     MSG_MULTI_NOTIF(prefix##_BLUE, defaultvalue, anncepre##_BLUE, infopre##_BLUE, centerpre##_BLUE) \
     MSG_MULTI_NOTIF(prefix##_YELLOW, defaultvalue, anncepre##YELLOW, infopre##YELLOW, centerpre##YELLOW) \
-    MSG_MULTI_NOTIF(prefix##_PINK, defaultvalue, anncepre##PINK, infopre##PINK, centerpre##PINK)
+    MSG_MULTI_NOTIF(prefix##_PINK, defaultvalue, anncepre##PINK, infopre##PINK, centerpre##PINK) \
+    MSG_MULTI_NOTIF(prefix##_GREEN, defaultvalue, anncepre##GREEN, infopre##GREEN, centerpre##GREEN)
 
 // MSG_MULTI_NOTIFICATIONS
     MSG_MULTI_NOTIF(COUNTDOWN_BEGIN,                    N_ENABLE,  ANNCE_BEGIN,    NULL,                                   CENTER_COUNTDOWN_BEGIN)
@@ -996,7 +1000,8 @@
     MSG_CHOICE_NOTIF_TEAM(NUM_TEAM_1, prefix##_RED, prefix, defaultvalue, challow, chtype, optiona##_RED, optionb##_RED) \
     MSG_CHOICE_NOTIF_TEAM(NUM_TEAM_2, prefix##_BLUE, prefix, defaultvalue, challow, chtype, optiona##_BLUE, optionb##_BLUE) \
     MSG_CHOICE_NOTIF_TEAM(NUM_TEAM_3, prefix##_YELLOW, prefix, defaultvalue, challow, chtype, optiona##_YELLOW, optionb##_YELLOW) \
-    MSG_CHOICE_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, challow, chtype, optiona##_PINK, optionb##_PINK)
+    MSG_CHOICE_NOTIF_TEAM(NUM_TEAM_4, prefix##_PINK, prefix, defaultvalue, challow, chtype, optiona##_PINK, optionb##_PINK) \
+    MSG_CHOICE_NOTIF_TEAM(NUM_TEAM_5, prefix##_GREEN, prefix, defaultvalue, challow, chtype, optiona##_GREEN, optionb##_GREEN)
 
 #undef N_DISABL
 #undef N_ENABLE
diff -ru /Applications/Xonotic/source/qcsrc/common/notifications/all.qh qcsrc/common/notifications/all.qh
--- /Applications/Xonotic/source/qcsrc/common/notifications/all.qh	2023-05-21 13:15:54
+++ qcsrc/common/notifications/all.qh	2026-08-28 01:33:29
@@ -81,7 +81,7 @@
 USING(Notification, entity);
 
 // used for notification system multi-team identifiers
-#define APP_TEAM_NUM(num, prefix) ((num == NUM_TEAM_1) ? prefix##_RED : ((num == NUM_TEAM_2) ? prefix##_BLUE : ((num == NUM_TEAM_3) ? prefix##_YELLOW : prefix##_PINK)))
+#define APP_TEAM_NUM(num, prefix) ((num == NUM_TEAM_1) ? prefix##_RED : ((num == NUM_TEAM_2) ? prefix##_BLUE : ((num == NUM_TEAM_3) ? prefix##_YELLOW : ((num == NUM_TEAM_4) ? prefix##_PINK : prefix##_GREEN))))
 #define APP_NUM(num, prefix) ((num) ? APP_TEAM_NUM(num, prefix) : prefix##_NEUTRAL)
 
 #define EIGHT_VARS_TO_VARARGS_VARLIST \
@@ -497,7 +497,8 @@
 		((f1 & BIT(0)) ? strcat(Team_ColoredFullName(NUM_TEAM_1), (f1 >> 1) ? ", " : "") : ""),
 		((f1 & BIT(1)) ? strcat(Team_ColoredFullName(NUM_TEAM_2), (f1 >> 2) ? ", " : "") : ""),
 		((f1 & BIT(2)) ? strcat(Team_ColoredFullName(NUM_TEAM_3), (f1 >> 3) ? ", " : "") : ""),
-		((f1 & BIT(3)) ?        Team_ColoredFullName(NUM_TEAM_4)                         : "")
+		((f1 & BIT(3)) ? strcat(Team_ColoredFullName(NUM_TEAM_4), (f1 >> 4) ? ", " : "") : ""),
+		((f1 & BIT(4)) ?        Team_ColoredFullName(NUM_TEAM_5)                         : "")
 	);
 }
 
@@ -855,7 +856,7 @@
 #define MSG_CHOICE_NOTIF_(teamnum, name, cvarname, defaultvalue, challow, chtype, optiona, optionb) \
 	REGISTER(Notifications, name, m_id, new_pure(msg_choice_notification)) { \
 		this.nent_choice_idx = nent_choice_count; \
-		if (!teamnum || teamnum == NUM_TEAM_4) \
+		if (!teamnum || teamnum == NUM_TEAM_5) \
 			nent_choice_count++; \
 		Create_Notification_Entity       (this, defaultvalue, ACVNN(cvarname), MSG_CHOICE, strtoupper(#name), teamnum); \
 		Create_Notification_Entity_Choice(this, ACVNN(cvarname), strtoupper(#name), \
diff -ru /Applications/Xonotic/source/qcsrc/common/teams.qh qcsrc/common/teams.qh
--- /Applications/Xonotic/source/qcsrc/common/teams.qh	2020-12-16 09:03:08
+++ qcsrc/common/teams.qh	2026-08-28 01:32:42
@@ -1,24 +1,27 @@
 #pragma once
 
-const int NUM_TEAMS = 4; ///< Number of teams in the game.
+const int NUM_TEAMS = 5; ///< Number of teams in the game.
 
 #ifdef TEAMNUMBERS_THAT_ARENT_STUPID
 const int NUM_TEAM_1 = 1; // red
 const int NUM_TEAM_2 = 2; // blue
 const int NUM_TEAM_3 = 3; // yellow
 const int NUM_TEAM_4 = 4; // pink
-const int NUM_SPECTATOR = 5;
+const int NUM_TEAM_5 = 5; // green
+const int NUM_SPECTATOR = 6;
 #else
 #ifdef CSQC
 const int NUM_TEAM_1 = 4; // red
 const int NUM_TEAM_2 = 13; // blue
 const int NUM_TEAM_3 = 12; // yellow
 const int NUM_TEAM_4 = 9; // pink
+const int NUM_TEAM_5 = 3; // green
 #else
 const int NUM_TEAM_1 = 5; // red
 const int NUM_TEAM_2 = 14; // blue
 const int NUM_TEAM_3 = 13; // yellow
 const int NUM_TEAM_4 = 10; // pink
+const int NUM_TEAM_5 = 4; // green
 #endif
 const int NUM_SPECTATOR = 1337;
 #endif
@@ -27,11 +30,13 @@
 const string COL_TEAM_2 = "^4";
 const string COL_TEAM_3 = "^3";
 const string COL_TEAM_4 = "^6";
+const string COL_TEAM_5 = "^2";
 // must be #defined, const globals drop the translation attribute
 #define NAME_TEAM_1 CTX(_("TEAM^Red"))
 #define NAME_TEAM_2 CTX(_("TEAM^Blue"))
 #define NAME_TEAM_3 CTX(_("TEAM^Yellow"))
 #define NAME_TEAM_4 CTX(_("TEAM^Pink"))
+#define NAME_TEAM_5 CTX(_("TEAM^Green"))
 #define NAME_TEAM _("Team")
 #define NAME_NEUTRAL _("Neutral")
 
@@ -40,20 +45,24 @@
 #define KEY_TEAM_2 CTX(_("KEY^Blue"))
 #define KEY_TEAM_3 CTX(_("KEY^Yellow"))
 #define KEY_TEAM_4 CTX(_("KEY^Pink"))
+#define KEY_TEAM_5 CTX(_("KEY^Green"))
 #define FLAG_TEAM_1 CTX(_("FLAG^Red"))
 #define FLAG_TEAM_2 CTX(_("FLAG^Blue"))
 #define FLAG_TEAM_3 CTX(_("FLAG^Yellow"))
 #define FLAG_TEAM_4 CTX(_("FLAG^Pink"))
+#define FLAG_TEAM_5 CTX(_("FLAG^Green"))
 #define GENERATOR_TEAM_1 CTX(_("GENERATOR^Red"))
 #define GENERATOR_TEAM_2 CTX(_("GENERATOR^Blue"))
 #define GENERATOR_TEAM_3 CTX(_("GENERATOR^Yellow"))
 #define GENERATOR_TEAM_4 CTX(_("GENERATOR^Pink"))
+#define GENERATOR_TEAM_5 CTX(_("GENERATOR^Green"))
 
 // used for replacement in filenames or such where the name CANNOT be allowed to be translated
 const string STATIC_NAME_TEAM_1 = "Red";
 const string STATIC_NAME_TEAM_2 = "Blue";
 const string STATIC_NAME_TEAM_3 = "Yellow";
 const string STATIC_NAME_TEAM_4 = "Pink";
+const string STATIC_NAME_TEAM_5 = "Green";
 
 #ifdef CSQC
 bool teamplay;
@@ -68,6 +77,7 @@
 		case NUM_TEAM_2: return COL_TEAM_2;
 		case NUM_TEAM_3: return COL_TEAM_3;
 		case NUM_TEAM_4: return COL_TEAM_4;
+		case NUM_TEAM_5: return COL_TEAM_5;
 	}
 
 	return "^7";
@@ -81,6 +91,7 @@
 		case NUM_TEAM_2: return '0.0625 0.0625 1'; // 0x0F0FFF
 		case NUM_TEAM_3: return '1 1 0.0625'; // 0xFFFF0F
 		case NUM_TEAM_4: return '1 0.0625 1'; // 0xFF0FFF
+		case NUM_TEAM_5: return '0.0625 1 0.0625'; // 0x0FFF0F
 	}
 
 	return '0 0 0';
@@ -94,6 +105,7 @@
 		case NUM_TEAM_2: return NAME_TEAM_2;
 		case NUM_TEAM_3: return NAME_TEAM_3;
 		case NUM_TEAM_4: return NAME_TEAM_4;
+		case NUM_TEAM_5: return NAME_TEAM_5;
 	}
 
 	return NAME_NEUTRAL;
@@ -108,6 +120,7 @@
 		case NUM_TEAM_2: return STATIC_NAME_TEAM_2;
 		case NUM_TEAM_3: return STATIC_NAME_TEAM_3;
 		case NUM_TEAM_4: return STATIC_NAME_TEAM_4;
+		case NUM_TEAM_5: return STATIC_NAME_TEAM_5;
 	}
 
 	return NAME_NEUTRAL;
@@ -121,6 +134,7 @@
 		case "blue": return NUM_TEAM_2;
 		case "yellow": return NUM_TEAM_3;
 		case "pink": return NUM_TEAM_4;
+		case "green": return NUM_TEAM_5;
 		case "auto": return 0;
 	}
 
@@ -138,6 +152,7 @@
 		case NUM_TEAM_2:
 		case NUM_TEAM_3:
 		case NUM_TEAM_4:
+		case NUM_TEAM_5:
 		{
 			return true;
 		}
@@ -156,6 +171,7 @@
 		case 2:
 		case 3:
 		case 4:
+		case 5:
 		{
 			return true;
 		}
@@ -174,6 +190,7 @@
 		case 2: return NUM_TEAM_2;
 		case 3: return NUM_TEAM_3;
 		case 4: return NUM_TEAM_4;
+		case 5: return NUM_TEAM_5;
 	}
 	return -1;
 }
@@ -189,6 +206,7 @@
 		case NUM_TEAM_2: return 2;
 		case NUM_TEAM_3: return 3;
 		case NUM_TEAM_4: return 4;
+		case NUM_TEAM_5: return 5;
 	}
 	return -1;
 }
Only in qcsrc: csprogs-mesh.pk3
diff -ru /Applications/Xonotic/source/qcsrc/server/scores_rules.qc qcsrc/server/scores_rules.qc
--- /Applications/Xonotic/source/qcsrc/server/scores_rules.qc	2023-05-10 11:55:32
+++ qcsrc/server/scores_rules.qc	2026-08-28 01:32:42
@@ -16,7 +16,7 @@
 
 int NumTeams(int teams)
 {
-	return boolean(teams & BIT(0)) + boolean(teams & BIT(1)) + boolean(teams & BIT(2)) + boolean(teams & BIT(3));
+	return boolean(teams & BIT(0)) + boolean(teams & BIT(1)) + boolean(teams & BIT(2)) + boolean(teams & BIT(3)) + boolean(teams & BIT(4));
 }
 
 // NOTE: ST_constants may not be >= MAX_TEAMSCORE
